pkgs/top-level/all-packages.nix

```diff
@@ -21304,6 +21304,8 @@ in
 
   tdrop = callPackage ../applications/misc/tdrop { };
 
+  tre-command = callPackage ../tools/system/tre {};
```

Comment:
Any reason for renaming this?

Reply:
yes, there's a existing library named `tre` here already.

Reply:
Ah, I see, can we rename the directory too in that case (pname can stay as it is)?

Reply:
done

Reply:
I think you need to change the path here too.

Reply:
Yup done